core_data_modules/traced_data/util/fold_traced_data.py

```diff
@@ -76,7 +76,7 @@ def assert_equal_keys_equal(td_1, td_2, equal_keys):
     def _is_missing_value(code):
         return code in {
             Codes.STOP, Codes.NOT_REVIEWED, Codes.NOT_INTERNALLY_CONSISTENT,
-            Codes.NOT_CODED, Codes.TRUE_MISSING, Codes.SKIPPED, None
+            Codes.NOT_CODED, Codes.TRUE_MISSING, Codes.SKIPPED, Codes.WRONG_SCHEME, None
```

Comment:
Are there any control codes that shouldn't be interpreted as missing?

Reply:
(I'm ok with this change as a quick fix)

Reply:
The problem here is with the function name, which has become less and less appropriate as our number of control codes has grown (and since we have gained the term "control code"). Will change to _is_control_code, which is appropriate given current uses of this (private) function